ee/audit/handler_test.go

```
@@ -0,0 +1,209 @@
+)
+
+type listAuditEventsWire struct {
+	Events     []map[string]any `json:"events"`
+	NextCursor *int64           `json:"nextCursor"`
+	Count      int64            `json:"count"`
+}
+
+func performList(t *testing.T, service *AuditService, query string) *httptest.ResponseRecorder {
+	t.Helper()
+	handler := NewAuditHandler(service)
+	request := httptest.NewRequest(http.MethodGet, "/api/audit/events?"+query, nil)
```

golangci-lint (2.12.2)

[error] net/http/httptest.NewRequest must not be called. use net/http/httptest.NewRequestWithContext

(noctx)